pkg/coin/coin.go

```diff
@@ -38,3 +38,7 @@ func GetAzetaDecFromAmountInZeta(zetaAmount string) (sdkmath.LegacyDec, error) {
 func (c CoinType) SupportsRefund() bool {
 	return c == CoinType_ERC20 || c == CoinType_Gas || c == CoinType_Zeta
 }
+
+func (c CoinType) IsAsset() bool {
```

Comment:
Add comment please
"represent transport of asset"

Reply:
https://github.com/zeta-chain/node/pull/3551/commits/6596863f4b5045e05799cfd06e5e40543ca2c90f